### PR TITLE
Charts part 1

### PR DESCRIPTION
**Pull request by tiananguyen**
Add a basic template and JS code for the users. The chart itself is currently filled with dummy Disney Movie data. Once our theme is figured out, the correct data will be implemented.

### CODE REVIEW

File: src/main/webapp/chart.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Charts Project(Part 1)</title>
+    <!-- take in core charts package -->
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script type="text/javascript">
+      google.charts.load('current', {packages: ['corechart']});
```
Comment (maksymko) [inline src/main/webapp/chart.html]: This script needs to live in its own file, similar to what is done for user-page. Put the code into src/webapp/js/charts-page-loader.js and include it here. Again, see user-page.html for the example of how to do this.

File: src/main/webapp/chart.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      google.charts.setOnLoadCallback(drawChart);
+      function drawChart(){
+        // create a new DataTable instance
+        var book_data = new google.visualization.DataTable();
+
+        // define columns for the DataTable instance
```
Comment (maksymko) [inline src/main/webapp/chart.html]: Right now you are loading dummy data, AFAICT. Please make this explicit here in the comment and add TODO for fixing this and loading data at runtime from somewhere, like this:

// Fill the table with static example data
// TODO: Load and update the data at runtime.

**Review by maksymko — CHANGES_REQUESTED** [review]
Tiana, you need to provide more details in PR description and commit messages, especially in the commit like this.
For example, you need to make clear that right now you are filling in the chart with dummy hard coded data.
Example of a commit message for this can be:

------ 8< ----------------8<-------------
Add chart page

Add basic template and JS code for the charts page. The chart itself is currently filled with static dummy data [what kind of data].

File: src/main/webapp/chart.html (hunk 2 above)
Comment (tiananguyen) [reply src/main/webapp/chart.html]: Added the TODO comment

File: src/main/webapp/chart.html (hunk 1 above)
Comment (tiananguyen) [reply src/main/webapp/chart.html]: Put the script in a separate javascript file

**Review by alyssa-lau — APPROVED** [review]
This looks good to go - just need to integrate changes with master

File: src/main/webapp/js/chart-loader.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+/*global google*/
```
Comment (maksymko) [inline src/main/webapp/js/chart-loader.js:1]: Google job figuring this one out :+1:

**Comment by maksymko** [comment]
This needs to be rebased on top of master, otherwise looks good.

**Review by maksymko — APPROVED** [review]
Looks good, thank you.

**Pull request merged**